lib/money/money.rb

```diff
@@ -390,11 +406,27 @@ def symbol
     currency.symbol || "¤"
   end
 
+  def infinite_precision?
+    !!@infinite_precision
```

Comment:
Do the `!!` operation during initialization, instead of every time this method is called.

Reply:
makes sense